backend/app/services/user_info.rb

```diff
@@ -26,7 +43,74 @@ def self.for_uuids(uuids)
     user_uuids
   end
 
+  def self.for_uuid(uuid)
+    return dev_user_info(uuid) unless Rails.env.production?
+
+    for_uuids([uuid]).values[0]&.table || {}
+  end
+
   def self.query_accounts(query)
     OpenStax::Accounts::Api.search_accounts(query).response.body
   end
+
+  def self.dev_researchers
+    Researcher.all.map do |researcher|
+      {
+        user_id: researcher.user_id,
+        first_name: researcher.first_name,
+        last_name: researcher.last_name,
+        name: "#{researcher.first_name} #{researcher.last_name}",
+        isResearcher: true,
+        isAdmin: false
+      }
+    end
+  end
+
+  def self.dev_admins
+    Admin.all.map do |admin|
+      {
+        user_id: admin.user_id,
+        first_name: 'Admin',
+        last_name: 'McAdminFace',
+        name: 'Admin McAdminFace',
+        isAdmin: true,
+        isResearcher: false
+      }
+    end
+  end
+
+  def self.find_user(uuid)
+    u = nil
+    mock_user = UserInfo.mock_users.find { |user| user[:user_id] == uuid }
+    researcher = dev_researchers.find { |r| r[:user_id] == uuid }
+    admin = dev_admins.find { |a| a[:user_id] == uuid }
+
+    if mock_user
+      u = mock_user
+    elsif researcher
+      u = researcher
+    elsif admin
+      u = admin
+    end
+
+    u
+  end
```

Comment:
with the above changes, I think this could be:
```
return UserInfo::MOCK_USERS[uuid] || 
   [Researcher, Admin].find{ |Klass| Klass.find_by(uuid: uuid)&.to_json(methods: [:is_admin, :is_researcher]) }
```